powershell/VstsTaskSdk/ToolFunctions.ps1

```diff
@@ -61,7 +61,9 @@ function Invoke-TaskScript {
         [scriptblock]$ScriptBlock)
 
     try {
+        $global:ErrorActionPreference = 'Stop'
         Write-Verbose "$($MyInvocation.MyCommand.Module.Name) $($MyInvocation.MyCommand.Module.Version)" 4>&1 | Out-Default
+        Initialize-SecureInputs 4>&1 | Out-Default
```

Comment:
Out-Default isn't fully hijacked yet at this point. I added a very small amount of verbose tracing within Initialize-SecureInputs so it needs to be explicitly piped to the lib's version of Out-Default.